Storage/Bookmarks/BookmarksModel.swift

```diff
@@ -159,7 +159,13 @@ public class MemoryBookmarkFolder: BookmarkFolder, SequenceType {
         }
 
         public mutating func next() -> BookmarkNode? {
-            return index < children.count ? children[index++] : nil
+            if index < children.count {
```

Comment:
Nit: Want to double check - Should this return children[index+ 1] or just children[index] then increment after?